Sources/ProjectDescription/EnvironmentVariable.swift

```diff
@@ -10,12 +10,16 @@ public struct EnvironmentVariable: Equatable, Codable, Hashable, ExpressibleBySt
     public var isEnabled: Bool
 
     // MARK: - Init
-
-    public init(value: String, isEnabled: Bool) {
+    
+    init(value: String, isEnabled: Bool) {
         self.value = value
         self.isEnabled = isEnabled
     }
```

Comment:
Can't we just remove this?

Reply:
No, if you define an `init(stringLiteral:)`, you will lose the init generated by the compiler

Reply:
You are right! No strong opinion, I would define that in an extension, so we have the generated one for free.